### PR TITLE
Prepare for 2.7 release

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,36 @@
...
 
 - Dropped support for php 5.5
 
-###
+### Deprecated
 
-- Phpunit 6 compatibility
+The following classes and endpoints were deprecated by Github and are also deprecated in the client: 
+
+- `Github\Api\Repo`
```
Comment (acrobat) [inline CHANGELOG.md]: `Github\Api\Repo::find()`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,15 +2,36 @@
...
+- `Github\Api\User`
```
Comment (acrobat) [inline CHANGELOG.md]: `Github\Api\User::find()`

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,15 +2,36 @@
...
+- `Github\Api\Issue::all()`
```
Comment (acrobat) [inline CHANGELOG.md]: `Github\Api\Issue::find()`

File: CHANGELOG.md (hunk 1 above)
Comment (Nyholm) [reply CHANGELOG.md]: The whole class is deprecated..

**Comment by Nyholm** [comment]
Oh, you are correct. Sorry

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,15 +2,36 @@
...
 
-###
+### Deprecated
 
-- Phpunit 6 compatibility
+The following endpoints were deprecated by Github and are also deprecated in the client: 
+
+- `Github\Api\Repo::find()`
+- `Github\Api\User::find()`
+- `Github\Api\Issue::all()`
```
Comment (acrobat) [inline CHANGELOG.md]: This also has to be the `find()` method instead of `all()`

**Comment by Nyholm** [comment]
There. Im done with this PR now

**Comment by Nyholm** [comment]
Thank you for the reviews

**Pull request merged**